docs/docs/0.3.2/policies/traffic-trace.md

```diff
@@ -0,0 +1,90 @@
+# Traffic Trace
+
+With the `TrafficTrace` policy you can configure tracing on every Kuma DP that belongs to the `Mesh`.
+Note that tracing operates on L7 HTTP traffic, so make sure that selected dataplanes are configured with HTTP Protocol.
+
+You can configure tracing in 3 steps:
+
+1) Configure tracing backend
+
+On Universal:
```

Comment:
I think, it would make sense to have separate lists of steps for `Universal` and `Kubernetes`.

Having them intermixed makes it harder to follow.

Reply:
But then we need to duplicate the tip and the 3rd step. How can we structure it then?

Reply:
up to @subnetmarco

Reply:
I'll merge it for now just to have docs for newest version. We can revise it in a separate PR.